react/components/SKUSelector/components/Variation.tsx

```diff
@@ -75,7 +75,7 @@ const Variation: FC<Props> = ({
     },
   } = useProduct()
 
-  const displayImage = isColor(name)
+  const displayImage = isColor(originalName)
```

Comment:
lol, can we delete the colors translation array now?

Reply:
Nope, because we still depend on the language it was written in the Catalog.